wpengine/atlas-content-modeler
# MTKA-1435 Email input validation

## Description

Add validation for input type Email.

[MTKA-1435](https://wpengine.atlassian.net/browse/MTKA-1435)

## Testing

1. Create a new model using the Email input.
2. Attempt to publish using new model with Email input types.
3. Verify email validation says it is required when email field is required.
4. Verify email validation says it needs to be a valid email address.

## Screenshots
<img width="1037" alt="Screen Shot 2022-03-25 at 11 31 36 AM" src="https://user-images.githubusercontent.com/62450648/160162316-ead1be11-f75b-4651-95ef-d4f393605486.png">
<img width="1042" alt="Screen Shot 2022-03-25 at 11 31 50 AM" src="https://user-images.githubusercontent.com/62450648/160162330-fed5e255-6de9-40b3-8a54-fabd0fa09e32.png">
<img width="1034" alt="Screen Shot 2022-03-25 at 11 32 04 AM" src="https://user-images.githubusercontent.com/62450648/160162353-296ec48d-c9bb-4254-abf3-f9f9a74ff908.png">
